db/schema.rb

```diff
@@ -30,6 +30,7 @@
     t.string   "title"
     t.text     "content_md"
     t.text     "content_html"
+    t.float  "price"
```

Comment:
that was changed by the previous migrations and probably changed later by hand.
Unfortunately both ways would break everyone else' s databases and models.